static/index.html

```diff
@@ -22,14 +22,19 @@ <h2 class="display-6 fs-2 mb-2">Connecting PrusaSlicer/SuperSlicer</h2>
 						<li class="list-group-item">Click "Gear"⚙️ (Add/Edit Physical Printer)</li>
 						<li class="list-group-item">Fill in "Descriptive name for the printer" with whatever you like</li>
 						<li class="list-group-item">Select your Ankermake printer profile from the dropdown menu</li>
+<<<<<<< HEAD
```

Comment:
Dude, you committed a merge conflict :fearful: 

That's... not good. Please rework this :)